src/lib/components/particles/map-locator/index.jsx

```diff
@@ -0,0 +1,23 @@
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const MapLocator = ({ svgPath, size, marker, styles }) => {
```

Comment:
I think `svgPath` should probably be called something like `basemapImage`. Arguably the projection used by the image should be passed as a prop? So that the component can figure out where the circle needs to be placed based on its lat/long.